machine_management/cluster_api_machine_management/cluster_api_provider_configurations/cluster-api-config-options-azure.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cluster-api-config-options-azure"]
+= Cluster API configuration options for {azure-full}
+include::_attributes/common-attributes.adoc[]
+:context: cluster-api-config-options-azure
+
+toc::[]
+
+:FeatureName: Managing machines with the Cluster API
+include::snippets/technology-preview.adoc[]
+
+You can change the configuration of your {azure-first} Cluster API machines by updating values in the Cluster API custom resource manifests.
+
+[id="cluster-api-sample-yaml-azure_{context}"]
+== Sample YAML for configuring {azure-full} clusters
+
+The following example YAML files show configurations for an {azure-full} cluster.
+
+//Sample YAML for a CAPI Azure infrastructure resource
+include::modules/capi-yaml-infrastructure-azure.adoc[leveloffset=+2]
+
+//Sample YAML for CAPI Azure machine template resource
+include::modules/capi-yaml-machine-template-azure.adoc[leveloffset=+2]
+
+//Sample YAML for a CAPI Azure compute machine set resource
+include::modules/capi-yaml-machine-set-azure.adoc[leveloffset=+2]
+
+// [id="cluster-api-supported-features-azure_{context}"]
+// == Enabling {azure-full} features with the Cluster API
+
+// You can enable the following features by updating values in the Cluster API custom resource manifests.
```

Comment:
Is this section present on any other platform? Does not sound like anything we support.

Reply:
yeah, it's commented out for potential future use in all platforms.